src/kOS.Safe/Screen/ScreenBuffer.cs

```diff
@@ -38,6 +44,12 @@ public class ScreenBuffer : IScreenBuffer
         public int RowCount { get; private set; }
         public int ColumnCount { get; private set; }
 
+        /// <summary>Delegate prototype expected by AddResizeNotifier</summary>
+        /// <param name="sb">This screenbuffer telling the callback who it is</param>
+        /// <returns>telling this screenbuffer how many vertical rows to scroll as a result of the resize.</returns>
```

Comment:
This return value isn't implemented in any of the delegates yet (they all just hardcode a return of zero), but the intent is that the delegate can tell the scrollbuffer to scroll itself based upon the SetSize.  This is how we might in the future implement the ability to keep the cursor in view when the commandline subbuffer is made narrower (and thus lines wrap onto more rows, moving the cursor out of view past the end of the terminal.  Right now that isn't fixed until the user types something, and the type-something code forces a scroll to bring the cursor into view.